app/assets/stylesheets/shared/_postcode-lookup.scss

```
@@ -2,4 +2,8 @@
   padding: govuk-spacing(3);
   margin: govuk-spacing(6) 0;
   background: govuk-colour("light-grey");
+
+  .govuk-hint .gem-c-govspeak {
+    color: govuk-colour("dark-grey");
```

I'd suggest using `$govuk-secondary-text-colour` - it'll have the same output, but means that if the secondary text colour changes in a future release of the Design System then this will keep up with that change as well.

```suggestion
    color: $govuk-secondary-text-colour;
```